config/secrets.yml

```diff
@@ -0,0 +1,14 @@
+default:: &default
+    mailer_domain: <%= ENV["MAILER_DOMAIN"] %>
+    mailer_password: <%= ENV["MAILER_PASSWORD"] %>
+    mailer_user: <%= ENV["MAILER_USER"] %>
+    mailer_address: <%= ENV["MAILER_ADDRESS"] %>
+    mailer_port: <%= ENV["MAILER_PORT"] %>
+    mailer_auth: <%= ENV["MAILER_AUTH"] %>
+    mailer_email: <%= ENV["MAILER_EMAIL"] %>
+development:
+    <<: *default
+production:
+    <<: *default
+test:
+    <<: *default
```

Comment:
🤔  this file should already be created at the start of the project. You need to at least have the `secret_key_base` here
You can set it up like this

development:
&nbsp;  secret_key_base: xxxxxxx

test:
&nbsp;  secret_key_base: xxxxxxx

production:
&nbsp;  secret_key_base: <%= ENV["SECRET_KEY_BASE"] %>

You can read about it here for example https://medium.com/@michaeljcoyne/understanding-the-secret-key-base-in-ruby-on-rails-ce2f6f9968a1